### PR TITLE
Add prowlarr extension

### PR DESCRIPTION
**Pull request by Elcaten**
## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by Elcaten** [comment]
An extension for [https://prowlarr.com](Prowlarr), an indexer manager/proxy built on the popular arr stack.
Contains a subset of functionality, available on the WEB.
Allows to to search for the torrents, copy download links and add to download clients.

File: extensions/prowlarr/src/search/SearchResultActionPanel.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  item,
+  onAction,
+}: {
+  item: ReleaseResource;
+  onAction: (params: { action: "addToDownloadClient"; item: ReleaseResource } | { action: "toggleDetails" }) => void;
+}) {
+  return (
+    <ActionPanel>
+      <Action
+        title="Add To Download Client"
```
Comment (pernielsentikaer) [inline extensions/prowlarr/src/search/SearchResultActionPanel.tsx:14]:
```suggestion
        title="Add to Download Client"
```

File: extensions/prowlarr/src/search/utils.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+  // return bytes if less than a KB
+  if (bytes < kiloBytes) return bytes + " Bytes";
+  // return KB if less than a MB
+  else if (bytes < megaBytes) return (bytes / kiloBytes).toFixed(decimal) + " KB";
+  // return MB if less than a GB
+  else if (bytes < gigaBytes) return (bytes / megaBytes).toFixed(decimal) + " MB";
+  // return GB if less than a TB
+  else return (bytes / gigaBytes).toFixed(decimal) + " GB";
+}
```
Comment (pernielsentikaer) [inline extensions/prowlarr/src/search/utils.tsx:19]: You can maybe use something like this
```ts
function formatBytes(bytes: number, precision: number = 1): string {
  const units = ["Bytes", "KB", "MB", "GB", "TB", "PB", "EB", "ZB", "YB"];
  if (bytes === 0) return "0 Bytes";

  const k = 1024;
  const i = Math.floor(Math.log(bytes) / Math.log(k));

  return parseFloat((bytes / Math.pow(k, i)).toFixed(precision)) + " " + units[i];
}
```

Comment (pernielsentikaer) [reply extensions/prowlarr/src/search/utils.tsx:19]: But I'm not sure it makes much sense

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you:

- Remove prowlarr-1.png from metadata since it doesn’t bring any contextual value

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

File: extensions/prowlarr/src/search/utils.tsx (hunk 2 above)
Comment (Elcaten) [reply extensions/prowlarr/src/search/utils.tsx]: Thanks! Looks more concise

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**